lib/rust/prelude/src/void.rs

```diff
@@ -0,0 +1,23 @@
+//! A small utility allowing writing nicer looking code.
+
+
+
+// ============
+// === Void ===
+// ============
+
+/// A small utility allowing writing nicer looking code. Consider the following example:
+///
+/// ```text
+/// let node = self.new_node_with(|node| {
+//      init(node);
+//  });
+/// ```
+/// 
+/// It is impossible to write it in a more compact form, as the `new_node_with` requires a closure
+/// which does not return a value. With this utility, the code can be rewritten to:
+/// ```text
```

Comment:
It _is_ possible, look:
```rust
let node = self.new_node_with(|node| { init(node); });
```
It even has fewer characters!

I guess you meant it's impossible _with our formatter_.

Also, maybe it just me, but I'd like to have rather something like `ToUnion` trait, to write 
```rust
let node = self.new_node_with(|node| init(node).to_union() );
```
As I like having as few nested () as possible.

Reply:
From call `.void()`